hip/base/hiprand_bindings.hip.hpp

```diff
@@ -72,11 +72,13 @@ struct is_supported<std::complex<double>> : std::true_type {};
 
 
 inline hiprandGenerator_t rand_generator(int64 seed,
-                                         hiprandRngType generator_type)
+                                         hiprandRngType generator_type,
+                                         hipStream_t stream)
 {
     hiprandGenerator_t gen;
-    hiprandCreateGenerator(&gen, generator_type);
-    hiprandSetPseudoRandomGeneratorSeed(gen, seed);
+    GKO_ASSERT_NO_HIPRAND_ERRORS(hiprandCreateGenerator(&gen, generator_type));
+    GKO_ASSERT_NO_HIPRAND_ERRORS(
```

Comment:
hiprandSetStream is available from 1.5.0

Reply:
which packaged HIP version does that coincide with?

Reply:
I think it should be available in all our considered version?